langchain4j/src/main/java/dev/langchain4j/service/V.java

```diff
@@ -27,7 +27,8 @@
  * String chat(@V String name, @V int age);
  * </pre>
  * <p>
- * When using Spring Boot, defining the value of this annotation is not required.
+ * When the application code has been built with the Java compiler's {@code -parameters} flag enabled
```

Comment:
Please read the comment above, it it already mentioned there. Spring Boot case was mentioned here additionally because of it's popularity (I doubt many users of Spring Boot know that `-parameters` is enabled implicitly).

Reply:
Cool, I'll remove this commit